NEWS.md

```diff
@@ -1,6 +1,9 @@
 
 # 1.1.0
 
+* Accept browser, HTTPS, or SSH URL in GitHub repo specification,
```

Comment:
CAn you please move this up to a new section. Version 1.1.0.9000? Thanks!

Reply:
1.1.0 is actually the current version....

Reply:
ok done!